src/emmodel/model.py

```diff
@@ -130,8 +132,12 @@ def _predict(self,
         results = self.results if results is None else results
         for i in range(self.num_models):
             self.data[i].attach_df(df)
+            if isinstance(results[i], dict):
```

Comment:
We could change the interface to let the input be `coefs` as well.

Reply:
I have made some changes to naming as suggested and would merge.